coreapi/codecs/corejson.py

```diff
@@ -33,19 +33,23 @@
 
 def encode_schema_to_corejson(schema):
     type_id = SCHEMA_CLASS_TO_TYPE_ID.get(schema.__class__, 'anything')
-    return {
+    retval = {
         '_type': type_id,
         'title': schema.title,
         'description': schema.description
     }
+    if isinstance(schema, coreschema.Enum):
+        retval['extra'] = {'enum': schema.enum}
```

Comment:
I think we could just put this value directly under `enum` rather than having an `extra` dictionary.

Thanks for progressing this!